lib/albacore/project.rb

```diff
@@ -210,6 +210,33 @@ def to_s
       path
     end
 
+    # Get Property folder path
+    def property_folder_path
+      read_property('AppDesignerFolder')
```

Comment:
If it doesn't exist, what then? You'll be File.join-ing with nil down below...

Suggestion; have a sane fallback; like the root of the project.